src/templates/company/index.js

```diff
@@ -17,6 +17,24 @@ function completeDomain(domain) {
   return domain ? `${domain}.aragonid.eth` : ''
 }
 
+function adjustVotingSettings(support, quorum) {
```

Comment:
I had a thought that maybe we should put this logic into the VotingScreen (and the similar logic for the TokenScreen)... but then decided that it might make more sense to keep the logic in each template since they know better than the VotingScreen what their limits are.